tests/v2_validation/cattlevalidationtest/core/test_ldap_auth.py

```diff
@@ -431,6 +477,10 @@ def test_ldap_create_new_env_add_group_member(admin_client):
         project.setmembers(members=new_members)
     assert "object has no attribute" in str(excinfo.value)
 
+    unset_group_search_domain(admin_client)
+    assert len(main_client.list_identity(name=group)) == 0
+    set_group_search_domain(admin_client)
+
```

Comment:
So we are changing the default search base (API_AUTH_LDAP_SEARCH_BASE) for all tests in this case. 

Instead lets just test the "groupSearchDomain" feature in a separate test by so that all other tests work as they worked before.
So in your test: 
first restrict the (API_AUTH_LDAP_SEARCH_BASE to ou=devs), then assert that group search fails)
Then set API_AUTH_LDAP_GROUP_SEARCH_BASE to the config, and assert that group search works.
At the end unset API_AUTH_LDAP_GROUP_SEARCH_BASE and set the API_AUTH_LDAP_SEARCH_BASE back to the usual.